projects/ids-enterprise-ng/src/lib/tree/soho-tree.component.ts

```diff
@@ -166,6 +166,12 @@ export class SohoTreeComponent implements AfterViewInit, OnInit, OnDestroy {
    * */
   @Output() selected = new EventEmitter<SohoTreeEvent>();
 
+  /**
+   * This event is fired when a node is unselected, the SohoTreeNode
+   * unselected is passed in the argument passed to the handler.
+   * */
+  @Output() deselected = new EventEmitter<SohoTreeEvent>();
```

Comment:
ALthough the name deselected might be slighly better i think we should just make this consistent and call it unselected like the actual EP event name?